### ISSUE TITLE
Merge: Project Scope

### ISSUE THREAD

**Issue by neatonk** [cwdevteam/mesa#49]
## Overview

This is a tracking issue covering tasks related to the code merge project, to be executed by @Kunkka0822 with assistance from @neatonk. Recent features have been developer in a separate codebase and will be merged into this repo, one feature at a time. Each feature should be implemented on a new feature branch, with a PR to merge the feature to the `dev` branch.

This will take place in parallel to the #45 project. The `dev` branch may periodically be rebased as needed to incorporate changes from `main`.

## Features

- [ ] #50
- [ ] #51
- [ ] Merge: Email invitations and notifications
- [ ] Merge: Contract builder and docusign integration
